modules/jupyter-widget/src/index.js

```diff
@@ -38,6 +43,10 @@ module.exports = {
   MODULE_NAME,
   TransportModel,
   TransportView,
+  // TODO PR reviewers, is there another way to test these
+  // functions aside from exporting them?
```

Comment:
Can we define a special `TEST_EXPORTS` object so we make it super clear which exports are only used by tests?

```
  TransportView,
  TEST_EXPORTS: {
     jsonConverter,
  }
```

Reply:
It seems the strategy is to export the function within the file but not within the index.js of the module.